### ISSUE TITLE
Select what groups that you actually want it turned on for

### ISSUE THREAD

**Issue by p0sty** [davidvavra/MoodSync#8]
My whole house is set up for LIFX. 

Turning it on changed the whole house which was not clear at all when I launched, which unintentionally annoyed my wife who was preparing dinner. ;)

could we please get the detection of lights and groups more clear cut - because it's not always appropriate for every LIFX.

**Comment by davidvavra** [comment]
Yes my app is pretty simple - triggers all the lights. You can send a PR for a light selection, it shouldn't be too hard.

**Comment by ackfool** [comment]
+1 This would be awesome. Only thing missing from the app. Any progress with this?

**Comment by davidvavra** [comment]
@ackfool As I said I don't actively develop this, but I will accept PRs.

**Comment by Royal2000H** [comment]
It seems there is one person who has worked on the app most recently and that's @johnwang16 https://github.com/johnwang16/MoodSync

Since he created a preferences menu, it seems the first step is already there for extending to group/bulb selection, if anyone is tempted to work on it.
